config/config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
@manangouhari can the whole config folder be deleted, now that's not needed.

Reply:
Yes, it can be deleted. I'll delete it and push the changes.